src/hotspot/cpu/aarch64/javaFrameAnchor_aarch64.hpp

```diff
@@ -54,9 +56,10 @@
     // To act like previous version (pd_cache_state) don't null _last_Java_sp
     // unless the value is changing
     //
+    // No fencing required, the members are declared volatile so the compiler will not reorder and
+    // the profiler always reads from the same thread and should observe the state in program order.
```

Comment:
I don't think copy() is every called.  Can we remove it?

Reply:
I missed that because I had commented out the JavaFrameAnchor(JavaFrameAnchor *src) ctor, thinking it was the only use, and it still builds.

I tried to research what this comment is about:
    // Hack Alert: Temporary bugfix for 4717480/4721647
and my conclusion is that it seems to be left over from an earlier safepoint implementation where the VM thread could look at the anchor of other threads while they were still running.  I don't think we do that anymore.